client_plugin/utils/fs/fs_linux.go

```diff
@@ -354,3 +356,27 @@ func getDevicePath(volDev *VolumeDevSpec) (string, error) {
 	}
 	return fmt.Sprintf("/dev/disk/by-path/pci-%s.0-scsi-0:0:%s:0", string(buf), volDev.Unit), nil
 }
+
+// GetMountInfo returns a map of mounted volumes and devices.
+func GetMountInfo(mountRoot string) (map[string]string, error) {
+	volumeMountMap := make(map[string]string) // map [volume mount path] -> device
+	data, err := ioutil.ReadFile(linuxMountsFile)
+
+	if err != nil {
+		log.Errorf("Can't get info from %s (%v)", linuxMountsFile, err)
+		return volumeMountMap, err
+	}
```

Comment:
Should we log the data for easier debugging, same as we have done in fs_windows.go?